.claude/skills/hns-lsel-applier/verify.sh

```
@@ -0,0 +1,191 @@
+  # First attempt failed (timeout OR non-timeout): retry exactly once.
+  run_with_timeout "$VERIFY_CMD"; ATTEMPTS=$((ATTEMPTS+1))
+  if [[ "$LAST_CLASS" == "success" ]]; then
+    OUTCOME=true
+  else
+    OUTCOME=false
+  fi
+fi
+
+append_ledger() {
+  # $1 verified  $2 attempts  $3 last_class
+  printf '{"stage":"verify","proposal_id":"%s","verified":%s,"attempts":%s,"last_class":"%s","ts":"%s"}\n' \
```

Shellcheck (0.11.0)

[warning] This format string has 5 variables, but is passed 4 arguments.

(SC2183)